src/Components/ServiceScene/ActionBarScene.tsx

```diff
@@ -39,6 +41,10 @@ export class ActionBarScene extends SceneObjectBase<ActionBarSceneState> {
       <Box paddingY={0}>
         <div className={styles.actions}>
           <Stack gap={1}>
+            {
+              // @ts-ignore appSidecar not yet in stable runtime
```

Comment:
Not sure how to deal with runtime updates. Should they be part of this or is it better to wait for a release and do something like this in the meantime?

Reply:
If you mean having to add the ts-whatever directive, it's okay. We also do that for other features that haven't been released https://github.com/grafana/explore-logs/blob/main/src/Components/ServiceScene/LogsPanelScene.tsx#L122